src/react-components/banner/Banner.js

```diff
@@ -0,0 +1,349 @@
+    defaultValues: {
+      email_format: "html"
+    }
+  });
+
+  /**
+   * On Form Error
+   */
+  const newsletterError = () => {
+    setResponseStatus(false);
+    setSubmitted(true);
+  };
+
+  /**
+   * On Form Success
+   */
+  const newsletterSuccess = () => {
+    setResponseStatus(true);
+    setSubmitted(true);
+  };
+
+  /**
+   * On AJAX Resp
+   * @param {Basket Response} resp
+   * @returns
+   */
+  const handleResponse = resp => {
+    const { status, statusText } = resp;
+
+    if (status !== 200 || status > 300 || status < 200) {
+      newsletterError();
+      return;
+    }
+
+    if (statusText !== "OK") {
+      newsletterError();
+      return;
+    }
+
+    // Do success stuff here..
+    newsletterSuccess();
+  };
+
+  const onSubmit = async data => {
+    const url = "https://basket.mozilla.org/news/subscribe/";
+    const body =
+      "email=" +
+      encodeURIComponent(data.email) +
+      "&format=" +
+      encodeURIComponent(data.email_format) +
+      "&newsletters=" +
+      "hubs" +
+      "&lang=" +
+      encodeURIComponent(navigator.language) +
+      "&source_url=" +
+      encodeURIComponent(window.location.origin);
+
+    try {
+      fetch(url, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/x-www-form-urlencoded"
+        },
+        body: body
+      }).then(handleResponse);
+    } catch (error) {
+      console.error(error);
+      newsletterError();
+    }
+  };
+
+  /**
+   * Emain input change
+   */
+  const onChangeEmail = useCallback(
+    e => {
+      setEmail(e.target.value);
+    },
+    [setEmail]
+  );
+
+  /**
+   * Checkbox Confirm
+   */
+  const onConfirm = () => {
+    setConfirm(state => !state);
+  };
+
+  /**
+   * Checkbox Label
+   * @returns JSX
+   */
+  const Label = () => {
+    return (
+      <>
+        I&apos;m okay with Mozilla handling my info as explained in this{" "}
+        <a href="https://www.mozilla.org/en-US/privacy/websites/" rel="noopener noreferrer" target="_blank">
+          Privacy Notice
+        </a>
+      </>
+    );
+  };
+
+  /**
+   * Post Submit Message
+   * @param {Boolean} param
+   * @returns JSX
+   */
+  const Messaging = ({ status }) => {
+    return <>{status ? <Success /> : <Error />}</>;
+  };
+
+  /**
+   * Success Message
+   * @returns JSX
+   */
+  const Success = () => {
+    return (
+      <div className={styles.message_wrapper}>
+        <img
+          src={cone}
+          className={styles.success_image}
+          alt={<FormattedMessage defaultMessage="celebrate" id="banner.celebrate-alt" />}
+        />
+        <div>
+          <h3 className={styles.message_title}>
+            <FormattedMessage defaultMessage="You're on the list" id="banner.success-title" />
+          </h3>
+          <p className={styles.message_body}>
+            <FormattedMessage
+              defaultMessage="Keep an eye out for product updates and an invite to join us as a tester in August."
```

Comment:
Talked with @emclaren - can we drop "in August" and just go "Keep an eye out for product updates and an invite to join us as a tester"?